docs/modules/user-guide/partials/proc_adding-components-to-a-devfile.adoc

```diff
@@ -94,10 +109,10 @@ Each component in a single devfile must have a unique name and use one of the ob
 |secure
 |boolean
 |no
-|Choose to define the endpoint as secure or not.
+|Choose whether or not to define the endpoint as secure.
```

Comment:
```suggestion
|Choose whether to define the endpoint as secure.
```
IBM SG: whether or not conjunction
Use "whether or not" only to mean "regardless of whether" (for example, "Log shunting runs whether or not log archiving is enabled"). Otherwise, use "whether".

Reply:
Hi @jc-berger. It looks like this issue is still unresolved.